### DISCUSSION TITLE
Change Wazuh Dashboard Title (HTML)

### DISCUSSION THREAD

**Discussion by zeroneaov** [wazuh/wazuh#18571] [General]
Dear All,

Can we actually change the window title of wazuh dashboard? By default, it is set to be Wazuh. Thank you.

**Comment by mdiego92** [comment]
Hello @zeroneaov,

To customize the title please edit your /etc/wazuh-dashboard/opensearch_dashboards.yml by adding the following line:

`opensearchDashboards.branding.applicationTitle: 'Title'`


Please remember to restart the service after saving the file:

`systemctl restart wazuh-dashboard`


Documentación for reference: https://opensearch.org/docs/1.2/dashboards/branding/

Hope this works for you!
Looking forward to your feedback!

**Reply by matooshak** [reply]
Hello @mdiego92 could you help with changing the navigation menu name of wazuh ? so that it doesnt show in the title as "Wazuh - Title" ?

**Reply by venomousgladiator** [reply]
please have u found out any solution

**Reply by aliicybersec** [reply]
I'm also looking for a solution to this problem. Has anyone found a solution?
